src/components/ArticleDetails/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import './styles.css';
+
+const border = {
+  borderColor: '#f5f5f5'
+};
+
+const ArticleContainer = ({ data, loading, error }) => {
```

Comment:
This is the Dumb Component not the container
the container is the one that will contain this one and make the api call 
you don't need to make it now though.